gcp/appengine/frontend3/src/templates/vulnerability.html

```diff
@@ -60,17 +60,31 @@ <h2 class="title">Affected packages</h2>
   <spicy-sections
       class="vulnerability-packages{% if vulnerability.affected|should_collapse %} force-collapse{% endif %}">
     {% for affected in vulnerability.affected %}
+      {% if affected['package']['ecosystem'] %}
```

Comment:
Would it make more sense to have a tab that just says "Source-only" or display the repo URL instead to keep things more consistent? 

This seems particularly important if there are other tabs for other ecosystems to render here in the same entry. I.e. if there is an Alpine tab.

Reply:
Addressed in [cd34ab9](https://github.com/google/osv.dev/pull/1447/commits/cd34ab994bd4fb1b57e5c01a49b9a503e32f58ea)